### ISSUE TITLE
Dropdown Check box checked icon is not visible in High contrast black/white mode.

### ISSUE THREAD

**Issue by nadavWeisler** [microsoft/fluentui#16161]
### Environment Information

- **Package version(s)**:  7.115.1
- **Browser and OS versions**: Edge chromium Version 86.0.601.1 (Official build) dev (64-bit) 

### Describe the issue:

Check box checked icon is not visible in High contrast black/white mode

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/nadavWeisler/pen/BaLzoBZ

#### Actual behavior:

Check box checked icon is not visible in High contrast black/white mode

#### Expected behavior:

Check box checked icon should be visible in High contrast black/white mode

### Documentation describing expected behavior

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b672c8a8b-ed4f-463f-868f-e8131caf9b74%7d

**Comment by paulgildea** [comment]
@nadavWeisler Thanks for filing this issue.

@ecraig12345 Can you determine if this repros with the current version of the library and root cause?

@xugao FYI

**Comment by dawolff-ms** [comment]
@paulgildea @ecraig12345 has there been any progress on this? Our team has a blocking MAS bug for this issue. 

Does anyone know of any workarounds?

**Comment by smhigley** [comment]
I actually can't reproduce the issue, I can see the checks OK (unless I'm missing something):
![screenshot of open multiselect dropdown, showing both checked and unchecked options](https://user-images.githubusercontent.com/3819570/113233442-19a3b780-9254-11eb-973b-d0baef3c6a9e.png)

**Comment by ecraig12345** [comment]
@dawolff-ms Which Fabric/Fluent version and Edge version are you seeing the issue in?

**Comment by dawolff-ms** [comment]
@ecraig12345 @smhigley 

Microsoft Windows 10 Enterprise Version 20H2 (OS build: 19042.804)
Chromium Edge Version 90.0.803.0 (Official build) dev (64-bit)
fluentui/react: 7.147.1

In the words of the MAS tester who logged this bug: "user is not able to identify [check marks] as check boxes, so the check box outline should be visible to the user. And also, keyboard focus should be clearly visible".

**Comment by dawolff-ms** [comment]
@smhigley if the checkmark looks good to you, then that's great. But, there is still an issue related to #12121 where in high contrast if all items are selected it becomes very difficult for the user to determine where keyboard focus is.

Do you think it's worth logging a separate issue for high contrast? Can this issue be closed if you don't see an issue with the checkmark?
